content/04.model-complexity.md

```diff
@@ -0,0 +1,35 @@
+### Techniques and procedures must be implemented to manage model complexity without sacrificing the value of machine learning
+Inherent challenges posed by low sample numbers in rare diseases are further aggravated by disease heterogeneity, poorly defined disease phenotypes, and often a lack of control (i.e. normal) data. 
+Machine learning approaches must be carefully designed to address these challenges. 
+We discuss how to implement methodological solutions like bootstrapping sample data, regularization methods for deep learning, and hyper-ensemble techniques to minimize misinterpretation of the data. 
+
+#### Bootstrapping
+
+Bootstrap or resampling computation is a powerful statistical technique that can be used for estimating population values from datasets of limited sample size by resampling the data to generate a distribution of the population statistic and minimize estimation error [@doi:10.1080/01621459.1997.10474007].
+Bootstrap based techniques are used in conjunction with various learning methods to find the most informative models given a specific dataset (e.g. bootstrap aggregating or bagging used in random forests [@doi:10.1023/A:1010933404324; @doi:10.1198/0003130043277], bootstrap in neural networks [@doi:10/c8xpqz], or regression models [@doi:10.1016/j.neucom.2004.11.017; @doi:10.1002/sim.4780111607]).
+In addition to model selection, rare disease datasets also necessitate the use of bootstrap to form an informative dataset [@doi:10.3390/genes11020226].
```

Comment:
Not sure what this means, exactly.

Reply:
My intent was to say that bootstrap can be used for model selection. Additionally it can also be used to enhance a small dataset. I will rephrase this line